src/update_agent/actor.rs

```diff
@@ -193,8 +203,20 @@ impl UpdateAgent {
             })
             .map(|res, actor, _ctx| {
                 match res {
-                    Some(release) => actor.state.update_available(release),
-                    None => actor.state.no_new_update(),
+                    Some(release) => {
+                        update_unit_status(&format!("update available: {}", release.version));
+                        actor.state.update_available(release);
+                    }
+                    None => {
+                        actor.state.no_new_update();
+                        update_unit_status(&format!(
+                            "periodically polling for updates (last checked {})",
+                            &actor
+                                .state_changed
+                                .with_timezone(&Local)
```

Comment:
Hmm yeah I hadn't considered old/stale TZ data. I was thinking that this is more informational and for convenience, so it'd be nice if the time were more easily readable by the admin. (I originally tried to make it show something like "last checked x min ago", but I guess that can't be very easily done here. 
But UTC works for me :)

Reply:
Personally, I find human-relative timestamps much more useful. UTC makes sense to start though ideally it'd be nice if we printed both (see e.g. https://github.com/coreos/rpm-ostree/blob/536bfc62ebb4d44b0126bed4e5050a08291614be/src/app/rpmostree-builtin-status.cxx#L1216).

Reply:
Since currently we only update the unit status when we change states (instead of in a loop like rpm-ostree) and we don't know when users will call `systemctl status zincati.service` (unlike `rpm-ostree status`), is it still possible to do human-relative timestamps? 
Could we make rpm-ostree builtin-status extract the time from the StatusText of zincati.service and then compute the human-relative time when `rpm-ostree status` is called? ...but we also can't guarantee that all potential update drivers output their time in the same format with their StatusText.

Reply:
Yeah you're right, the architecture doesn't allow it very easily. I guess we could shove the timestamp as a u64 in a separate custom `sd_notify` variable that rpm-ostree picks up on, but it's not worth the complexity IMO.

Maybe we could print in both UTC and local time? But anyway, just UTC is fine for now!